protocol/0056-REWA-rewards_overview.md

```diff
@@ -980,6 +1006,16 @@ At the end of epoch 2, 10000 VEGA rewards should be distributed to the `ETHUSDT`
 - If an eligible party has net relative returns strictly greater than `0` over the last `window_length` epochs, their returns volatility reward metric should equal the variance of their relative returns over the last `window_length` epochs (<a name="0056-REWA-089" href="#0056-REWA-089">0056-REWA-089</a>).
 - If an eligible party has net relative returns strictly greater than `0` over the last `window_length` epochs in multiple in-scope markets, their return volatility reward metric should be the variance of their relative returns in each market (<a name="0056-REWA-090" href="#0056-REWA-090">0056-REWA-090</a>).
 
+### Realised returns
+
+- If an eligible party has a non-profitable position which has not been closed, they will not have a realised returns score and should receive no rewards (<a name="0056-REWA-109" href="#0056-REWA-109">0056-REWA-109</a>).
```

Comment:
maybe I'm missing it but I don't see how that works with the above formulas. $m_{rz}$ will be non zero if either the party closes a position *or* is paying/receiving funding payment/M2M so why in the case above will they nor have a score?